src/test/java/org/jabref/logic/importer/fileformat/BibtexParserTest.java

```diff
@@ -2238,4 +2239,29 @@ void parseInvalidBibDeskFilesResultsInWarnings() throws IOException {
 
         assertEquals(List.of(firstEntry, secondEntry), result.getDatabase().getEntries());
     }
+
+    @Test
+    void parseCommentToJson() {
+        String entries =
+                """
+                    @Comment{jabref-meta-0.1.0
+                        {
+                            "saveActions" :
+                            {
+                                "state": true
+                            }
+                        }
+                """;
+        BibtexParser parser = new BibtexParser(importFormatPreferences);
+        Optional<JsonObject> actualJson = parser.parseCommentToJson(entries);
```

Comment:
I am not sure if this will be the real API, but it is OK for me to let this through.